components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -1826,6 +1829,28 @@ private void translateInformation(Element root) throws FormatException {
         }
       }
 
+      Element sNode = getFirstNode(dimensions, "S");
+      if (sNode != null) {
+        NodeList scenes = sNode.getElementsByTagName("Scene");
+        int nextPosition = 0;
+        for (int i=0; i<scenes.getLength(); i++) {
+          Element scene = (Element) scenes.item(i);
+          NodeList positions = scene.getElementsByTagName("Position");
+          for (int p=0; p<positions.getLength(); p++) {
+            Element position = (Element) positions.item(p);
+            String x = position.getAttribute("X");
+            String y = position.getAttribute("Y");
+            String z = position.getAttribute("Z");
+            if (nextPosition < positionsX.length && positionsX[nextPosition] == null) {
+              positionsX[nextPosition] = new Length(DataTools.parseDouble(x), UNITS.MICROM);
+              positionsY[nextPosition] = new Length(DataTools.parseDouble(y), UNITS.MICROM);
+              positionsZ[nextPosition] = new Length(DataTools.parseDouble(z), UNITS.MICROM);
+              nextPosition++;
```

Comment:
Sorry, yes, those two files were missed by accident.  Configurations for both should be fixed with https://github.com/openmicroscopy/data_repo_config/pull/220/commits/8bde52c329cab924098483463b20c1abe0793f53